.bazelci/presubmit.yml

```diff
@@ -136,7 +136,6 @@ tasks:
     platform: ubuntu2004
   mac_workspace:
     <<: *reusable_config
-    <<: *common_workspace_flags
```

Comment:
Actually not, I was a bit tired and swept this under the rug :(. The docs targets are failing in the WORKSPACE mode and I don't know how to fix it. Spent a lot of time investigating. You can see the failure here: https://buildkite.com/bazel/rules-python-python/builds/9779#01934a4c-6b6d-417f-bc1f-e03576b83a60

It's completely reporducible on Ubuntu builds, but it's not failing on the CI, because WORKSPACE mode isn't properly enabled on the Ubuntu. Let me do another PR to enable it.

Since I don't know how to fix it, I can remove docs targets from the Workspace CI builds if you're OK with that.

Reply:
Enabling workspace on CI: https://github.com/bazelbuild/rules_python/pull/2435

Reply:
Fixed